src/reference/asciidoc/whats-new.adoc

```diff
@@ -15,6 +15,12 @@ If you are interested in more details, see the Issue Tracker tickets that were r
 [[x5.3-new-components]]
 === New Components
 
+[[x5.2-rsocket-support]]
+==== Integration Pattern
```

Comment:
5.3 - will fix on merge after travis is green.

Reply:
Doh... And not `rsocket-support` 😄 

Thanks

Reply:
Also not rsocket - fixed.